src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+
+<body>
+    <h1 class="title">Leaderboard</h1>
+    <div class="container">
+        <div class="scoreCard">
+            
+        </div>
+
+        <form action="" class="inputCard">
+            <input type="text" name="" id="user" placeholder="Your name" required>
+            <input type="number" name="" id="score" placeholder="Your score" required>
+            <button type="submit" id="submit">Submit</button>
+        </form>
+    </div>
+</body>
+
+</html>
```

Comment:
- [ ] As requested in the requirements, let's kindly make sure that the HTML codes look like in the proposed wireframe. The headers **Recent scores**, **Add your scores**, and the button **Refresh** are missing on your HTML. The form placeholders `Your name`, and `Your score` is not visible. Please let's kindly fix it


| Wireframe :+1:  | Your design :thinking:  |
| --- | --- |
| ![Screenshot from 2023-03-07 15-35-59](https://user-images.githubusercontent.com/63087491/223456529-6cdf7599-3258-49d0-8a3f-659b168531df.png) | ![Screenshot from 2023-03-07 15-36-39](https://user-images.githubusercontent.com/63087491/223456639-5a316d23-9d91-4881-abcf-b760b475e11b.png) |